src/main/webapp/app/communication/posting-reactions-bar/posting-reactions-bar.component.html

```diff
@@ -42,6 +42,15 @@
         }
     }
 
+    @if (showSearchResultInAnswersHint()) {
+        <div class="search-result-in-answers-info">
+            <fa-icon class="fa-xs align-self-center" [icon]="faInfoCircle" />
+            <span class="emoji-count px-2">
+                {{ 'artemisApp.metis.overview.searchResultInAnswersHint' | artemisTranslate }}
+            </span>
```

Comment:
We generally should use the jhi-translate instead of the translate pipe https://docs.artemis.cit.tum.de/dev/guidelines/client.html#localization